msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -295,6 +295,64 @@ The JSON response body for Meeting Details API is as follows:
     }
 } 
 ```
+
+## CART API 
+
+The communication access real-time translation (CART) API enables your app to expose a POST endpoint for Microsoft Teams CART captions.
```

Comment:
This paragraph is a bit misleading. Applications can send captions to the CART API to send the captions directly to an ongoing Teams meeting. Let's change this to

```
The CART API exposes a POST endpoint for Microsoft Teams CART captions – human-typed closed captions. Text content sent to this endpoint will be displayed to end users in a Microsoft Teams meeting when they have captions enabled. 
```